react-app/src/index.css

```
@@ -10,7 +76,9 @@
 }
 
 :root {
-  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
+  --font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
```

it would be better if you make the variable more specific see suggested changes below
```suggestion
  --font-family-base: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
  --font-family-title: system-ui, Avenir, Helvetica, Arial, sans-serif;
```